src/Apps/Artist/Routes/CV/ArtistCVRoute.tsx

```diff
@@ -42,14 +46,20 @@ export const ArtistCVRouteFragmentContainer = createFragmentContainer(
   {
     viewer: graphql`
       fragment ArtistCVRoute_viewer on Viewer {
-        soloShows: artist(id: $artistID) {
+        soloShows: artist(id: $artistID)
+          @principalField
```

Comment:
Does this work? I thought MP will loudly complain if you pass in a query with more than one `@principalField` directive declared (that's sort-of implied with the 'principal' naming -> can be only one).

Reply:
fixed!

Reply:
Sweet - thanks!